src/pudl/workspace/zenodo_download.py

```diff
@@ -0,0 +1,336 @@
+#!/usr/bin/env python
```

Comment:
Was this module superceded by the code that got integrated into the datastore module?